pkg/cable/libreswan/libreswan.go

```diff
@@ -87,18 +87,18 @@ func NewLibreswan(localEndpoint types.SubmarinerEndpoint, localCluster types.Sub
 		return nil, errors.Errorf("error parsing CR_IPSEC_NATTPORT environment variable")
 	}
 
-	nattPort, err := localEndpoint.Spec.GetBackendPort(subv1.UDPPortConfig, int32(defaultNATTPort))
+	NATTPort, err := localEndpoint.Spec.GetBackendPort(subv1.UDPPortConfig, int32(defaultNATTPort))
```

Comment:
We shouldn't capitalize local vars. We only capitalize initialisations when it's not the first word. Same in other places.